onos-umbrella/templates/raftcluster.yaml

```diff
@@ -1,21 +1,20 @@
-# SPDX-FileCopyrightText: 2022 2020-present Open Networking Foundation <info@opennetworking.org>
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
 
-{{- if .Values.atomix.store.consensus.enabled }}
-apiVersion: consensus.atomix.io/v1beta1
-kind: ConsensusStore
+
+apiVersion: raft.atomix.io/v1beta3
```

Comment:
We need to have the `.Values.atomix.store.consensus.enabled` condition here, as well, so we can use this umbrella chart in internal umbrella charts.

Reply:
`{{- if .Values.atomix.store.consensus.enabled }}`

Reply:
nevermind we don't use the umbrella chart